getYear.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"errors"
+	"github.com/rwcarlsen/goexif/exif"
```

Comment:
I think you might need to take a look at your editor config — a properly configured `goimports` would create two import blocks separated by a newline, with standard library imports at the top and third-party imports at the bottom.